console/frontend/pom.xml

```diff
@@ -122,9 +122,9 @@
 								</pluginExecution>
 								<pluginExecution>
 									<pluginExecutionFilter>
-										<groupId>org.commonjava.maven.plugins</groupId>
+										<groupId>com.github.hazendaz.maven</groupId>
```

Comment:
Moved to a maintained version, from:
https://github.com/hazendaz/directory-maven-plugin/